### DISCUSSION TITLE
Listing jetons(DON or SVET)

### DISCUSSION THREAD

**Discussion by DevelanceOfficial** [tonmetaspace/.github#33] [Ideas]
#contribution

Add your token (don or light) to the listing so that money flows into it in return, we give pure bitcoin from mining.

We offer investors to pour money into a coin (don or light), the coin becomes liquid.
In return, we offer them to get pure bitcoin from mining farms.

**Comment by inozemtsev-roman** [comment]
# DEX
![image](https://user-images.githubusercontent.com/24755187/226099191-73fa6204-75cd-419e-a0d1-c610be19ae34.png)
[Telegram bot](https://t.me/tonmetaspace_bot)

## DON/SVET AMM-LP
https://github.com/tonmetaspace/dex/blob/master/src/services/api/addresses.ts#L42

[AMM LP Donation Contract](https://verifier.ton.org/EQAm79IxsggEDUBArrw3yXlbKmapvCj2rlzKNoj5tCsgMnZ1)

```ts
    "EQDEcJlTPBymzUqOJ15QR44vIlPIHhsWllrIafWpPdeHiuNR": {
        ammMinter: "EQAm79IxsggEDUBArrw3yXlbKmapvCj2rlzKNoj5tCsgMnZ1",
        tokenMinter: "EQDEcJlTPBymzUqOJ15QR44vIlPIHhsWllrIafWpPdeHiuNR",
        image: "https://cache.tonapi.io/imgproxy/FDcUw2JFUphF2irJPrZdQP-Uz9R2urTvJtvjAaXFDNE/rs:fill:200:200:1/g:no/aHR0cHM6Ly9pcGZzLmlvL2lwZnMvYmFma3JlaWhwcW5ramNmd2ZpcGVrZ2o3Z3RwY2hiNmFkbm83czZ5ZGlnb25iaHN2d3hkbWV4NWF3YXU.webp",
        displayName: "DON",
        color: "#0061d2",
        name: "Donation",
        decimals: 9,
    },
```

[AMM LP Svet Contract](https://verifier.ton.org/EQDXjFixseiUM1Y2NMpxfRNEhfj7zHHe68qP1N9DCBrOo8sO)

```ts
    "EQAgkc5YfAA1Hc75QXcZJrL15l3LXoQr0CaK4sqcpHRFXFxh": {
        ammMinter: "EQDXjFixseiUM1Y2NMpxfRNEhfj7zHHe68qP1N9DCBrOo8sO",
        tokenMinter: "EQAgkc5YfAA1Hc75QXcZJrL15l3LXoQr0CaK4sqcpHRFXFxh",
        image: "https://cache.tonapi.io/imgproxy/BJDj9oHHl_JVNVc6c7x72g0Kxy98miugZO8vwZMed4Q/rs:fill:200:200:1/g:no/aHR0cHM6Ly9iYWZrcmVpY28zeGdyYWxidmUzNzRsN29zeWpiM2s3dXh1eWFrcWxzeGcyeTZpYW5veGcyaXJ0ZnZqeS5pcGZzLm5mdHN0b3JhZ2UubGluay8.webp",
        displayName: "Svet",
        color: "#390075",
        name: "TON Svet",
        decimals: 9,
    },
```

Source-code: https://github.com/tonmetaspace/dex/
DEX on gh-pages: https://tonmetaspace.github.io/dex/manage-liquidity/add-liquidity/EQDEcJlTPBymzUqOJ15QR44vIlPIHhsWllrIafWpPdeHiuNR
